backend/.gitignore

```diff
@@ -4,5 +4,6 @@
 /build/
 package-json.lock
 coverage
+log.txt
```

Comment:
we should place all log-files in the directory logs, because the whole logs directory is gitignored

Reply:
This log file I had created accidentally manual and didn't belong there at all. 
Yes you are right, all logfiles into log.
ok I will change it.